Dockerfile

```diff
@@ -1,11 +1,10 @@
 #Start with a base image containing Java runtime
 FROM openjdk:17-jdk-slim
 
-#Information around who maintains the image
-#MAINTAINER thietnguyen
+WORKDIR /app
```

Comment:
You can keep the maintainer name

Reply:
Thank for your feedback, I added maintainer back to Dockerfile